### ISSUE TITLE
Can't see search text or flag text in item spawning menu

### ISSUE THREAD

**Issue by terribleperson** [CleverRaven/Cataclysm-DDA#78157]
### Describe the bug

When spawning an item using the debug menu, the search bar appears to be invisible. You're still able to search, and even apply flags, but you can't see what you're doing.

### Attach save file

[Sangerville-trimmed.tar.gz](https://github.com/user-attachments/files/17909953/Sangerville-trimmed.tar.gz)

### Steps to reproduce

Use the debug menu to spawn an item.  Press / to search, search for an item. Observe the missing search bar. Press enter to finish searching, then press F to start flagging. Observe no text entry box. Type your flag and hit enter. Observe that the item is now flagged. 

### Expected behavior

Text entry should generally be visible.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.5131 (22H2)
- Game Version: bbc0d00 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro]
]

**Comment by moxian** [comment]
Same issue also happens to pocket settings.

This is because the debug spawn menu is implemented in imgui, and the filter is the old-style popup, which is being drawn "under" imgui.

Update your game. This should be "fixed" by #77985 .
And by "fixed" i mean the search prompt being faint but barely visible since imgui becomes slightly transparent. 

<details><summary>pic</summary>

![20241125-153735-cataclysm-tiles](https://github.com/user-attachments/assets/3a06f0a7-8f70-449d-9724-33a1ee3b7807)

</details>

You can then make it even more readable by putting something like
```json
{
  "//": "See doc/COLOR.md for documentation on how to use this file.",
  "inherit_base_colors": false,
  "colors": {
    "ImGuiCol_WindowBg"               : [0.06, 0.06, 0.06, 0.50]
  }
}
```

 into your `config/imgui_style.json` - the smaller the last number in the array (`0.50` in the example above) - the more readable it would be (not without side effects).

Long-term fix requires #77863

**Comment by RenechCDDA** [comment]
Duplicate of #77290
